pkgs/development/compilers/cudatoolkit/redist/extension.nix

```diff
@@ -0,0 +1,32 @@
+final: prev: let
+
+  inherit (final) callPackage;
+  inherit (prev) cudaVersion lib pkgs;
+
+  ### Cuda Toolkit Redist
+
+  # Manifest files for redist cudatoolkit. These can be found at
+  # https://developer.download.nvidia.com/compute/cuda/redist/
+  cudaToolkitRedistManifests = {
+    "11.4" = ./manifests/redistrib_11.4.4.json;
+    "11.5" = ./manifests/redistrib_11.5.2.json;
+    "11.6" = ./manifests/redistrib_11.6.2.json;
```

Comment:
IIUC we're not supporting redist packages for CUDA versions 11.0-11.3?

Reply:
Correct, those files don't exist.